File: src/Rectangle.java
public class Rectangle {
    private double width = 1.00;
    private double height = 1.00;

    public Rectangle (double width, double height){
        this.width = width;
        this.height = height;

    }

    public double getWidth() {
        return width;
    }
    public double getHeight(){
        return height;
    }
    public double getArea(){
        return height*width;
    }
    public double getPerimeter(){
        return (width+height) * 2;
    }
    public String printRectangle(String objectName, Rectangle r){
        return "Rectangle " + objectName + " is " + r.getWidth() + " units wide and " + getHeight() + " units high.";
    }
}

File: assignment12/RectangleApp/Rectangle.cs
public class Rectangle {
    private double width;
    private double height;

    public Rectangle(){
        width = 1.0;
        height = 1.0;
    }

    public Rectangle(double width, double height){
        this.width = width;
        this.height = height;
    }

    public double GetWidth(){
    return this.width;
    }

    public double GetHeight(){
        return this.height;
    }

    public double GetArea(){
        return this.width * this.height;
    }

    public double GetPerimeter(){
        return 2 * (this.width + this.height);
    }

    public void printRectangle(String objectName){
        Console.WriteLine( "Rectangle "+ objectName + " is " + this.width + " units wide and " + this.height + " units high.");
    }
}